### DISCUSSION TITLE
Unable to find eslint library

### DISCUSSION THREAD

**Discussion by ddiegow** [LazyVim/LazyVim#2329] [Q&A]
Hi, I recently installed lazyvim, added eslint-lsp via Mason and it shows as attached via LspInfo. But every time I open a file, I get the message `Warning: unable to find eslint library` I've looked online trying to figure out why it happens, but haven't found anything. Could somebody help me understand what I'm doing wrong?

**Comment by mikalcallahan** [comment]
@ddiegow do you have `eslint` installed either locally (in your workspace) or globally?

**Reply by heapifyman** [reply]
I get the same message even though eslint is installed globally.

```
   ~  eslint --version                                                                                
v8.56.0
```

But I do use `nvm` so the path to `eslint` is: `/home/<USER>/.nvm/versions/node/v20.11.0/bin/eslint`. Could that be a problem?
